src/main/java/org/iban4j/bban/BbanStructure.java

```diff
@@ -348,13 +359,26 @@ private BbanStructure(final BbanStructureEntry... entries) {
                 new BbanStructure(
                         BbanStructureEntry.bankCode(4, 'a'),
                         BbanStructureEntry.accountNumber(21, 'c')));
+        
+        structures.put(CountryCode.SC,
+                new BbanStructure(
+                		BbanStructureEntry.bankCode(4, 'a'),
+    					BbanStructureEntry.branchCode(4, 'n'),
+                        BbanStructureEntry.accountNumber(16, 'n'),
+                        BbanStructureEntry.accountType(3, 'a')));
```

Comment:
The problem is that the bankCode is actually `4!a2!n` and the branchCode is `2!n`, but we can't support the mixed representation.

Is it better to say bankCode is 6!c and the branchCode is 2!n?

Reply:
bank.code appears to agree with this code.